languages/python/oso/tests/test_polar_roles.py

```diff
@@ -139,6 +139,7 @@ def test_empty_role(init_oso):
         roles = {
             member: {}
         };
+    actor_has_role_for_resource(_, _, _) if False;
```

Comment:
Should be lower case `false`. `False` is unbound and evaluates to true...

Reply:
ah I always mess this up

Reply:
The return value doesn't actually matter--the rule just needs to be here so the validation error isn't thrown.

Reply:
But yes I'll fix it so it's not confusing.

Reply:
> `False` is unbound and evaluates to true...

![https://media.giphy.com/media/jUpbLiWxLrI6KNI4g3/giphy.gif](https://media.giphy.com/media/jUpbLiWxLrI6KNI4g3/giphy.gif)

Reply:
Actually it can't be `false` otherwise we won't recognize it when we conduct inline queries to check for it.

Reply:
@saolsen I laughed out loud when I saw that gif earlier today, then forgot about it, came back, and it got me again